Source/DataSources/PolygonGeometryUpdater.js

```diff
@@ -406,6 +406,10 @@ PolygonGeometryUpdater.prototype._setStaticOptions = function (
     Iso8601.MINIMUM_VALUE,
     ArcType.GEODESIC
   );
+  options.textureCoordinates = Property.getValueOrUndefined(
+    polygon.textureCoordinates,
+    Iso8601.MINIMUM_VALUE
+  );
```

Comment:
This file doesn't seem complete, e.g. `textureCoordinates` isn't declared in the constructor and there's other places where it might be missing.

Reply:
Thanks for catching this, I completely missed what's going on in this file - I've done a pass over the file and handled `textureCoordinates` like all the other properties.